src/utils.ts

```diff
@@ -7,3 +7,13 @@ export function getNumbers(from: number, to: number): number[] {
 
   return numbers;
 }
+
+export function getPaginatedItems(items: string[], perPage: number) {
+  const paginated = [];
+
+  for (let i = 1; i < Math.ceil(items.length / perPage) + 1; i++) {
+    paginated.push(items.slice(i * perPage - perPage, i * perPage));
+  }
+
+  return paginated;
+}
```

Comment:
The `getPaginatedItems` function lacks type annotations for its parameters and return type. It's good practice to use TypeScript's type system to its full extent to prevent potential bugs and improve code maintainability.